TKeazirian.HTTPServer.Tests/Response/ResponseTests.cs

```diff
@@ -64,11 +68,12 @@ public void GetBodyReturnsBody()
     {
         string responseStatus = "HTTP/1.1 200 OK";
         string responseBody = "Hello world";
+        byte[] responseBodyBytes = ByteConverter.ToByteArray(responseBody);
         string responseHeaders = HelperFunctions.CreateTestResponseHeaders(responseBody);
 
         Response response =
-            new Response(responseStatus, responseHeaders, responseBody);
+            new Response(responseStatus, responseHeaders, responseBodyBytes);
```

Comment:
why the `Bytes`? that's more of those implementation details leaking into the variable names.

Reply:
this is all over this PR, I think it is just more of [this](https://github.com/t-keazirian/HTTPServer/pull/36#discussion_r932346391)

Reply:
I fixed this - maybe you're looking at an older commit?

Reply:
maybe github is doing me a disservice. Right now I see 24 files changed.  I would expect far less once you go back to the non-bytes versions.  However I am not seeing it that way at the moment.

Reply:
I could pair at some point to sort it out.

Reply:
That would be great!

Reply:
Yeah, I'm seeing what you are seeing now - maybe because I changed it and then changed it back?  That is strange so def could use a quick pair to sort it out.  Thanks!